core/src/main/java/io/questdb/cairo/vm/MemoryPARWImpl.java

```diff
@@ -812,38 +886,10 @@ long getLongBytes(int page, long pageOffset, long pageSize) {
         return value;
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The result of `Integer` multiplication is cast into a larger type. If you think the result of the multiplication could exceed the size of an `Integer`, consider casting the arguments of the multiplication to avoid arithmetic overflows. For more information, see [CWE-190](https://cwe.mitre.org/data/definitions/190.html).